src/website/views/components/toolbar/_examples.pug

```diff
@@ -1,499 +1,1371 @@
-            <button class="chi-button -icon -flat" aria-label="Button action">
-              <div class="chi-button__content">
-                <i class="chi-icon icon-x"></i>
+            <div class="chi-toolbar__end">
+              <div class="chi-toolbar__actions">
+                <div class="chi-toolbar__actions-desktop">
+                  <button class="chi-button -portal -icon -primary -flat" id="button-pr-download" aria-label="Download" data-tooltip="Download" data-position="top">
+                    <div class="chi-button__content">
+                      <i class="chi-icon icon-arrow-to-bottom"></i>
+                    </div>
+                  </button>
+                  <button class="chi-button -portal -icon -primary -flat" id="button-pr-refresh" aria-label="Refresh" data-tooltip="Refresh" data-position="top">
+                    <div class="chi-button__content">
+                      <i class="chi-icon icon-refresh"></i>
+                    </div>
+                  </button>
+                  <button class="chi-button -portal -icon -primary -flat" id="button-pr-column-customization" aria-label="Column Customization" data-tooltip="Column Customization" data-position="top">
+                    <div class="chi-button__content">
+                      <i class="chi-icon icon-table-column-settings"></i>
+                    </div>
+                  </button>
+                </div>
+                <div class="chi-toolbar__actions-mobile">
+                  <button class="chi-button -portal -icon -primary -flat chi-drawer__trigger" id="drawer-trigger-brp2" data-target="#drawer-2" aria-label="Button action">
+                    <div class="chi-button__content">
+                      <i class="chi-icon icon-more-vert"></i>
+                    </div>
+                  </button>
+                </div>
               </div>
-            </button>
+            </div>
           </div>
-        </div>
-      </div>
-
-      <!-- Drawer -->
-      <div class="chi-backdrop -closed">
-        <div class="chi-backdrop__wrapper">
-          <div class="chi-drawer -left -menu -position--absolute" id="drawer-1">
-            <div class="chi-drawer__header">
-              <span class="chi-drawer__title">Filters</span>
-              <button class="chi-button -icon -close" aria-label="Close">
+          <div class="chi-toolbar__ribbon">
+            <div class="chi-toolbar__start">
+              <i class="chi-icon icon-arrow-down -xs"></i>
+              <div class="chi-toolbar__results">
+                <span class="chi-toolbar__label">Search Results </span>
+                <span>(1-40 of 78)</span>
+              </div>
+              <div class="chi-divider -vertical"></div>
+              <chi-link href="#" size="md">Save View</chi-link>
+            </div>
+            <div class="chi-toolbar__end">
+              <button class="chi-button -portal -icon -primary -flat" aria-label="Button action">
```

Comment:
-portal class is not needed anymmore, please remove

Reply:
Solved.